test/net/sourceforge/kolmafia/session/EncounterManagerTest.java

```diff
@@ -208,198 +209,247 @@ void underTheKnifeIsConditionalAutostop(boolean skipping) {
-    Preferences.setInteger("_saberForceMonsterCount", 3);
-    Preferences.setString("_saberForceMonster", monsterName);
+    @ParameterizedTest
+    @CsvSource({"Demoninja, true", "generic duck, false"})
+    void isSaberForceMonsterFromName(String monsterName, String expected) {
+      Preferences.setInteger("_saberForceMonsterCount", 3);
+      Preferences.setString("_saberForceMonster", monsterName);
 
-    boolean actual = EncounterManager.isSaberForceMonster(monsterName, "Pandamonium Slums");
+      boolean actual = EncounterManager.isSaberForceMonster(monsterName, "Pandamonium Slums");
 
-    assertThat(actual, equalTo(Boolean.parseBoolean(expected)));
-  }
+      assertThat(actual, equalTo(Boolean.parseBoolean(expected)));
+    }
 
-  @ParameterizedTest
-  @CsvSource({"Demoninja, true", "generic duck, false"})
-  void isSaberForceMonsterFromData(String monsterName, String expected) {
-    Preferences.setInteger("_saberForceMonsterCount", 3);
-    Preferences.setString("_saberForceMonster", monsterName);
+    @ParameterizedTest
+    @CsvSource({"Demoninja, true", "generic duck, false"})
+    void isSaberForceMonsterFromData(String monsterName, String expected) {
+      Preferences.setInteger("_saberForceMonsterCount", 3);
+      Preferences.setString("_saberForceMonster", monsterName);
 
-    var monster = MonsterDatabase.findMonster(monsterName);
-    boolean actual = EncounterManager.isSaberForceMonster(monster, "Pandamonium Slums");
+      var monster = MonsterDatabase.findMonster(monsterName);
+      boolean actual = EncounterManager.isSaberForceMonster(monster, "Pandamonium Slums");
 
-    assertThat(actual, equalTo(Boolean.parseBoolean(expected)));
+      assertThat(actual, equalTo(Boolean.parseBoolean(expected)));
+    }
+
+    @Test
+    void isSaberForceMonsterFromNextMonster() {
+      Preferences.setInteger("_saberForceMonsterCount", 3);
+      Preferences.setString("_saberForceMonster", "Demoninja");
+      var monster = MonsterDatabase.findMonster("Demoninja");
+      MonsterStatusTracker.setNextMonster(monster);
+
+      boolean actual = EncounterManager.isSaberForceMonster();
+
+      assertThat(actual, equalTo(true));
+    }
   }
 
-  @Test
-  void isSaberForceMonsterFromNextMonster() {
-    Preferences.setInteger("_saberForceMonsterCount", 3);
-    Preferences.setString("_saberForceMonster", "Demoninja");
-    var monster = MonsterDatabase.findMonster("Demoninja");
-    MonsterStatusTracker.setNextMonster(monster);
+  @Nested
+  class SpookyVHSTape {
+    @Test
+    void isSpookyVHSTapeEncounterBasedOnResponseText() {
+      String html = html("request/test_fight_spooky_vhs_tape_monster.html");
 
-    boolean actual = EncounterManager.isSaberForceMonster();
+      boolean actual = EncounterManager.isSpookyVHSTapeMonster(html, false);
 
-    assertThat(actual, equalTo(true));
+      assertThat(actual, equalTo(true));
+    }
+
+    @Test
+    void isSpookyVHSTapeEncoutnerBasedOnMonster() {
```

Comment:
Encounter, not Encoutner.

And how about Cleanups?

(I should retrofit my early tests to have Cleanups, but these days, all new tests should use them.)

Reply:
These are just copied and pasted from Enamorang support. I will update them both!